src/transactions/api.ts

```diff
@@ -1,27 +1,51 @@
-import { createApi } from '@reduxjs/toolkit/query/react'
-import { baseQuery } from 'src/redux/api'
-import type { TokenTransaction } from 'src/transactions/types'
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
+import { type LocalCurrencyCode } from 'src/localCurrency/consts'
+import {
+  NetworkId,
+  TokenTransactionTypeV2,
+  type PageInfo,
+  type TokenTransaction,
+} from 'src/transactions/types'
+import networkConfig from 'src/web3/networkConfig'
 
-export const FIRST_PAGE_TIMESTAMP = 0 // placeholder
+export const FIRST_PAGE_CURSOR = 'FIRST_PAGE'
 
 export type TransactionFeedV2Response = {
   transactions: TokenTransaction[]
-  pageInfo: {
-    hasNextPage: boolean
-  }
+  pageInfo: PageInfo
 }
 
+const ALLOWED_ZERION_NETWORKS: NetworkId[] = [
```

Comment:
@kathaypacific specific list of allowed networks is controlled by blockchain-api. `networkConfig.networkToNetworkId` for alfajores would return the following error:
```
Unexpected networkIds. Allowed networkIds are ethereum-mainnet,celo-mainnet,op-mainnet,base-mainnet,arbitrum-one,polygon-pos-mainnet
```

So there's a need to whether somehow sync up these lists whenever they change on API side or just move the logic for this specific settings to the API side, as per my comment above.

Reply:
@kathaypacific What I meant is `networkConfig` has two sets of settings: one for `Testnets.alfajores` and one for `Testnets.mainnet`. There is no valid settings we can set up for alfajores' `networkConfig.networkToNetworkId`.

Reply:
yes that's fine right? i think it's more expected that the wallet passes alfajores networkid's when running in alfajores, even if that results in an error from the api because the api is not supposed to work in alfajores. the best case scenario is that if zerion one day supports alfajores, we wouldn't need a client side change to enable the homefeed